pkg/upgrade/upgrade.go

```diff
@@ -615,3 +615,179 @@ func cleanupModelControllerLegacyDeployment(ctx context.Context, cli client.Clie
 
 	return nil
 }
+
+// TODO: to be removed: https://issues.redhat.com/browse/RHOAIENG-21080
+func PatchOdhDashboardConfig(ctx context.Context, cli client.Client) error {
+	log := logf.FromContext(ctx)
+
+	var dashboardConfigs unstructured.UnstructuredList
+	dashboardConfigs.SetGroupVersionKind(gvk.OdhDashboardConfig)
+
+	if err := cli.List(ctx, &dashboardConfigs); err != nil {
+		log.Error(err, "Failed to list odhdashboardconfig CRs")
+		return fmt.Errorf("error listing odhdashboardconfig CRs: %w", err)
+	}
+
+	for _, cr := range dashboardConfigs.Items {
+		if !cluster.IsNotReservedNamespace(&corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: cr.GetNamespace()}}) {
+			continue
+		}
+
+		log.Info("Found CR, applying patch", "namespace", cr.GetNamespace(), "name", cr.GetName())
+
+		patch := cr.DeepCopy()
+		updates := map[string][]any{
```

Comment:
nip: since the updates map is always the same, can't this be just a var ?

Reply:
var would do fine 😄